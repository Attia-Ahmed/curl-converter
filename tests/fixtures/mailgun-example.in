curl -s -X POST --user 'api:0987654321' https://api.mailgun.net/v3/example.com/messages -F from='Excited User <mailgun@YOUR_DOMAIN_NAME>' -F to[]=YOU@YOUR_DOMAIN_NAME -F to[]=bar@example.com -F subject='Hello' -F text='Testing some Mailgun awesomeness!'
