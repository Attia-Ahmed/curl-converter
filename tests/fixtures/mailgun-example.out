Http::asMultipart()
    ->withBasicAuth('api', '0987654321')
    ->post('https://api.mailgun.net/v3/example.com/messages', [
        'from' => 'Excited User <mailgun@YOUR_DOMAIN_NAME>',
        'to' => ['YOU@YOUR_DOMAIN_NAME', 'bar@example.com'],
        'subject' => 'Hello',
        'text' => 'Testing some Mailgun awesomeness!',
    ]);
